src/app/public/modules/text-editor/services/text-editor-adapter.service.ts

```diff
@@ -124,9 +120,7 @@ export class SkyTextEditorAdapterService {
         }
 
         /* istanbul ignore next */
```

Comment:
Do we still need to ignore istanbul?

Reply:
I'm seeing quite a few `/* istanbul ignore next */` spread throughout this file; any way to remove some of those? (I know one or two of them are for IE 11 stuff?)

Reply:
I was able to remove a few. I think it would do us some good to write some extra tests to cover some of the rest. I think this is just a side-effect of contribution. Would you be open to doing that work in another PR?